app/models/ems_refresh.rb

```diff
@@ -25,8 +25,8 @@ def self.debug_trace
   mattr_accessor :debug_failures
 
   # Development helper method for setting up the selector specs for VC
-  def self.init_console(use_vim_broker = false)
-    ManageIQ::Providers::Vmware::InfraManager::Refresher.init_console(use_vim_broker)
+  def self.init_console
+    ManageIQ::Providers::Vmware::InfraManager::Refresher.init_console
   end
```

Comment:
Not for this PR, but since this method is provider specific...I wonder if it should move over to the vmware provider somehow...not sure the best way to accomplish that.  @agrare @kbrock Thoughts?

Reply:
Yeah was this was supposed to be a general entry point for other providers' console specific code?  Just moving this to the vmware refresher sounds good to me.

Reply:
I was hoping to completely get rid of this.

Reply:
For this PR it doesn't matter... @kbrock I don't see how you can get rid of it...we need to preload the selector spec when the broker is not available.